torch/distributed/rpc/api.py

```diff
@@ -350,6 +351,42 @@ def _validate_rpc_args(backend, store, name, rank, world_size, rpc_backend_optio
             )
 
 
+T = TypeVar("T")
+GenericWithOneTypeVar = Generic[T]
+
+
+try:
+    class RRef(PyRRef, GenericWithOneTypeVar):
+        # Combine the implementation class and the type class.
+        pass
+except TypeError as exc:
+    # TypeError: metaclass conflict: the metaclass of a derived class
+    # must be a (non-strict) subclass of the metaclasses of all its bases
+    #
+    # This used to show in Python3.5 tests.
+    # Now it is stil seen in XLA tests.
+    import sys
+
+    raise AssertionError(
```

Comment:
when you raise here, shouldn't the following metaclass definition be invalid?

Reply:
Not meant for landing. Just want to see what Python version those failed tests are using. Will revert this piece before landing.